src/Dockerfile

```
@@ -240,6 +240,18 @@ ADD rootfs/files/scripts/install_rke2.sh /buildroot/files/scripts/install_rke2.s
 RUN --security=insecure /buildroot/files/scripts/install_rke2.sh
 ADD rootfs/files/configs/etc/super/var/lib/rancher/rke2/rke2-pss.yaml ${OUTPUTDIR}/etc/super/var/lib/rancher/rke2/rke2-pss.yaml
 
+# copy pki-authority service files
+ADD rootfs/files/scripts/install_lxc_deps.sh /buildroot/files/scripts/
```

Hadolint (2.14.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)